packages/ui-components/src/main.ts

```diff
@@ -4,8 +4,8 @@ import './exports/hero-component';
 export { ProfileButton } from './components/profile-button';
 import './exports/profile-button';
 
-export { EnableButton } from './components/enable-button'
-import './exports/enable-button'
+export { EnableButton } from './components/enable-button';
+import './exports/enable-button';
```

Comment:
Could we sort these automatically with eslint?

Reply:
Yes, but changing eslint config may be out of scope for the current PR